owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -884,14 +886,15 @@ public void onItemClick(AdapterView<?> l, View v, int position, long id) {
                 } else if (PreviewVideoFragment.canBePreviewed(file) &&
                         !fileIsDownloading(file)) {
                     // FIXME: 13/10/2020 : New_arch: Av.Offline
-//                    // Available offline exception, don't initialize streaming
-//                    if (!file.isAvailableLocally() && file.isAvailableOffline()) {
-//                        // sync file content, then open with external apps
-//                        ((FileDisplayActivity) mContainerActivity).startSyncThenOpen(file);
-//                    } else {
-//                        // media preview
-//                        ((FileDisplayActivity) mContainerActivity).startVideoPreview(file, 0);
-//                    }
+                    // Available offline exception, don't initialize streaming
+                    // if (!file.isAvailableLocally() && file.isAvailableOffline()) {
```

Comment:
Maybe now is not needed to control when isAvailableOffline, but check to be sure and commented code can be removed.

Reply:
Same here. A little polish will be done in upcoming PRs. This one won't be merged into master for the moment. Some refactors will be done before that moment 👍